libraries/src/MVC/View/CategoriesView.php

```diff
@@ -96,7 +96,7 @@ public function display($tpl = null)
             return false;
         }
 
-        if ($parent == false) {
+        if (!$parent) {
```

Comment:
$parent is a `CategoryNode` object or `null`, so better use strict comparison with null
```suggestion
        if ($parent === null) {
```